pyuvdata/uvdata/tests/test_ms.py

```diff
@@ -234,13 +269,20 @@ def test_multi_files(casa_uvfits, axis):
     testfile1 = os.path.join(DATA_PATH, "multi_1.ms")
     testfile2 = os.path.join(DATA_PATH, "multi_2.ms")
 
+    # It seems that these two files were made using the CASA importuvfits task, but the
+    # history tables are missing, so we can't infer that from the history. This means
+    # that the uvws are not flipped/data is not conjugated as they should be. Fix that.
+
     filesread = [testfile1, testfile2]
     # test once as list and once as an array
     if axis is None:
         filesread = np.array(filesread)
 
     uv_multi.read(filesread, axis=axis)
-    # Casa scrambles the history parameter. Replace for now.
+    # uv_multi.uvw_array *= -1
+    # uv_multi.data_array = np.conj(uv_multi.data_array)
```

Comment:
remove these commented lines. If they go with the prior comment maybe remove it too?

Reply:
Deleted.